initial_setup.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+# this script performs the initial setup
+
+URL="http://0.0.0.0:8000/"
+status_code=000
+ENV_EXAMPLE=".env.example"
+ENV_ORIG=".env"
+TIMEOUT_TIME=60
+
+error_check() {
+    if [ $? -ne 0 ]; then
+        echo "FAILED"
+        exit
+    fi
+}
+
+# instantiate .env file
+cp ${ENV_EXAMPLE} ${ENV_ORIG}
+
+# runs docker compose as daemon process
+docker-compose up -d
```

Comment:
I think it's not good to leave a daemon running after the one-time initialization is finished. Or, we could print a notice to STDOUT that a docker-compose daemon is left lingering.

Reply:
I think it's best not to start the docker-compose as a daemon. Starting it as a  foreground process will be more verbose which will be more helpful in case of any unexpected error.
So, maybe we can remove the `-d` option?

Reply:
Would you run it in the background with its output directed to stdout? Maybe just request that `docker-compose up` be already running for this script to proceed.